onnx/shape_inference/implementation.cc

```diff
@@ -473,6 +506,10 @@ std::vector<const TypeProto*> GraphInferencerImpl::doInferencing(
 
   return graphOutputTypes;
 }
-
+void deleteCreatedTensors(std::vector<TypeProto*> initializerTypeList) {
```

Comment:
Thanks for adding this! Minor nit: "deleteCreatedTypes" would be a better name than "deleteCreatedTensors".

Reply:
Good idea.